server/src/utils/validators.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SurveyResultRequest } from '../models/surveyModel';
+
+export const surveyResultRequestValidator = (requestData: SurveyResultRequest): string[] => {
+  const { callId, acsUserId, response } = requestData;
+  const errors: string[] = [];
+  const responseAllowedtypes = ['boolean', 'number', 'string'];
+
+  // Check fields are undefined
+  if (!callId) errors.push('callId is missing');
```

Comment:
instead of "is missing" can it be replaced with "must be present"

Reply:
That sounds better. I will update the code!